reporting/sending_metrics_report.md

```diff
@@ -0,0 +1,19 @@
+# Sending Metrics Report
+
+## Configurations
+
+Sending Metrics Report is an automated task in UgandaEMR, however it requires setting some facility Credentials ie. Analytics Username and Password and DHIS2 Facility UUID
+
+To set the Analytics User name and password,
+
+1. Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;UgandaemrSync.
+
+2. Set the Analytics Username and Password as METS and METS1234 respectively and save.
```

Comment:
@dbaluku we are using the value of the DHIS2 uuid for the facility as the username and password for the analytics 

>> add a screenshot

Reply:
Done